gtk4/Cargo.toml

```diff
@@ -25,6 +25,11 @@ v4_10 = ["ffi/v4_10", "v4_8"]
 xml_validation = ["gtk4-macros/xml_validation"]
 unsafe-assume-initialized = []
 
+# Versions from https://gitlab.gnome.org/GNOME/gnome-build-meta/-/tree/gnome-43/elements/sdk
+# gtk takes care of setting the versions of gsk and gdk
+# gio takes care of setting the version of glib
+gnome_43 = ["v4_8", "cairo-rs/v1_16", "gdk-pixbuf/v2_42", "gio/v2_74", "pango/v1_50"]
```

Comment:
what about glib?

Reply:
we should also look at adding 42 tbh, as it is commonly used still.

Reply:
> what about glib?

gio takes care of setting the version of glib. If I understand it correctly, gio is also part of GLib (as C library) or at least they share the same version. Since gio depends on glib, setting the version there also sets it for glib.

Alright, I'll add GNOME 42 too :)

Reply:
> gio takes care of setting the version of glib. If I understand it correctly, gio is also part of GLib (as C library) or at least they share the same version. Since gio depends on glib, setting the version there also sets it for glib.

Ah that is true, never mind my comment then.